accounts/abi/type.go

```diff
@@ -33,7 +33,7 @@ const (
 	FixedBytesTy
 	BytesTy
 	HashTy
-	RealTy
+	FixedPointTy
```

Comment:
I'd call this `Fixedpoint` (i.e. lowercase `p` since it's closer to `fixed-point` than `fixed point` (the latter of which is a completely different thing). It also makes it less similar to `FixedBytes`, where the `Fixed` part has a completely different meaning).